_posts/2020-07-10-removing-negative-words

```diff
@@ -0,0 +1,86 @@
+
+Moving away from the "master / slave" terminology, however, has seen more controversy. As such, the following sections will be dedicated to that particular change.
+
+As Matthew Ahrens eloquently explains in his [pull request](https://github.com/openzfs/zfs/pull/10435) removing all possible references to the term "slave" from OpenZFS' `zfs` repository:
+
+>The horrible effects of human slavery continue to impact society. The casual use of the term "slave" in computer software is an unnecessary reference to a painful human experience.
+
+Empathy in itself is a pretty good motivation for change and knowing that these terms are hurtful and can easily be replaced are sufficient to prompt us to act. However, we also believe in understanding why we are doing things and how what we are doing fits with our company culture and purpose. Simply following the trend without questioning why it's important, how it's beneficial and where it fits in a bigger picture context isn't enough.
+
+Understanding these points is paramount to create a culture of ambassadors of change. Questioning, done with an open mind and actual desire to understand, is healthy and desirable.
+
+### Why is a change in language needed?
+
+There is an intrinsic relationship between language and power. And there is an unstopable evolution in language. Every day, new terms arise and old terms become obsolete.
+
+The "master / slave" terminology is clearly connected to slavery. And slavery is far from being a concept "lost in time". First and foremost because the issue of human slavery should never be forgotten. And second because slavely is, sadly, far from being an issue that is no longer present in society.
+
+As outlined by IETF in its memo, in order to effectively communicate, it is important to make the information accessible to all readers, and this implies evaluating whether the words and terms used will hinder the reader and get in the way of effective communication. Additionally, if we're to not think of anything but clarity, the "master / slave" usage in the context of human slavery far predates its use in software and engineering. And there's no descriptive relationship between human slavery and software.
+
+Therefore, these terms should not be casually used in any context and should never lose the historical meaning they carry. There is no real necessity to use them in technology, as there are several other pairs that are just as descriptive without being offensive.
+
+### Why is this important?
+
+In a community where open source is very much a reality, that advocates and pioneers change and challenges established premises, the use of controversial terminology, specially done unnecessarily, is counter-intuitive.
```

Comment:
@ABizzinotto I think this sentence is a bit too long and hard to follow. Could you please break it down in two or three sentences? I get the meaning but I had to read it three times to understand it.

Reply:
Hmmm do you have any suggestions? What part is hard to follow? I made some changes to the structure of the sentence but I don't know how to break that without compromising the message I am building to.